samples/create-symbol-styles-from-web-styles/src/main/java/com/esri/arcgismaps/sample/createsymbolstylesfromwebstyles/screens/CreateSymbolStylesFromWebStylesScreen.kt

```diff
@@ -0,0 +1,123 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.createsymbolstylesfromwebstyles.screens
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Info
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.MapView
+import com.esri.arcgismaps.sample.createsymbolstylesfromwebstyles.components.CreateSymbolStylesFromWebStylesViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun CreateSymbolStylesFromWebStylesScreen(sampleName: String) {
+
+    // Get a reference to the view model
+    val mapViewModel: CreateSymbolStylesFromWebStylesViewModel = viewModel()
+
+    // Keep track of legend visibility state
+    var showLegend by remember { mutableStateOf(false) }
+
+    Scaffold(topBar = { SampleTopAppBar(title = sampleName) }, content = {
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(it),
+        ) {
+            MapView(
+                modifier = Modifier.fillMaxSize(), arcGISMap = mapViewModel.arcGISMap,
+                // Hide the legend on any tap of the map view
+                onDown = { showLegend = false },
+                // Update the map scale in the view model
+                onMapScaleChanged = mapViewModel::onMapScaleChanged
+            )
+            // Show the FAB
+            if (!showLegend) {
+                FloatingActionButton(modifier = Modifier
```

Comment:
Move the comment inside the `if` statement
```suggestion
            if (!showLegend) {
                // Show the FAB
                FloatingActionButton(modifier = Modifier
```